projects/plugins/wpcomsh/bin/create-github-release.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
```

Comment:
At the very least, let's stick something here to prevent people accidentally running this.
```suggestion

echo "Do not use this script anymore. Use the monorepo release tooling." >&2
exit 1
```